libraries/core-react/src/components/Combobox/Combobox.stories.tsx

```diff
@@ -0,0 +1,211 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { useState } from 'react'
+import { Combobox, ComboboxProps, Button, Typography, EdsProvider } from '../..'
+import { Story, Meta } from '@storybook/react/types-6-0'
+import { UseMultipleSelectionStateChange } from 'downshift'
+import styled from 'styled-components'
+import { action } from '@storybook/addon-actions'
+import { useForm, Controller } from 'react-hook-form'
+import { items } from '../../stories/data'
+
+export default {
+  title: 'Components/Combobox',
+  component: Combobox,
+  parameters: {
+    docs: {
+      description: {
+        component: `The Combobox component allows users to choose one or
+        multiple items or options from a list.
+        `,
+      },
+    },
+  },
+} as Meta
+
+const Wrapper = styled.div`
+  margin-bottom: 350px;
+`
+
+export const Default: Story<ComboboxProps> = (args) => (
+  <Wrapper>
+    <Combobox label="You can play with me" {...args} items={items} />
+  </Wrapper>
+)
+export const Multiple: Story<ComboboxProps> = (args) => (
+  <Wrapper>
+    <Combobox label="Single" {...args} items={items} />
+    <Combobox label="Multiple" {...args} items={items} multiple />
+  </Wrapper>
+)
+export const Disabled: Story<ComboboxProps> = () => (
+  <Combobox label="Choose an element" meta="km/t" items={items} disabled />
+)
+
+export const ReadOnly: Story<ComboboxProps> = () => (
+  <Combobox label="This is read only" items={items} readOnly></Combobox>
+)
+
+const initial = ['Troms og Finnmark', 'Vestland']
+
+export const WithPreselected: Story<ComboboxProps> = () => (
+  <Wrapper>
+    <Combobox
+      label="I have preselected options"
+      items={items}
+      initialSelectedItems={initial}
+    />
+  </Wrapper>
+)
+
+export const UseOnChangeHandler: Story<ComboboxProps> = () => {
```

Comment:
Storybook crashes when testing the combobox in this example

Reply:
Fixed!